tests/src/Functional/ReplicationSettingsTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Drupal\Tests\workspace\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\multiversion\Entity\Workspace;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\simpletest\BlockCreationTrait;
+use Drupal\workspace\Entity\WorkspacePointer;
+
+/**
+ * Tests replication settings with the internal replicator.
+ *
+ * @group workspace
+ */
+class ReplicationSettings extends BrowserTestBase {
```

Comment:
This test is intended as an alternative to the InternalReplicatorTest. This test is intended to be a functional test, creating a workspace with replication settings that then is tested for pushing/pulling to the parent (Live) workspace.

This test appears to fail due to workspaces not being moderatable. The cause may be simply how I am running the command, as such I am copying and pasting my full command and output:

```
vagrant@sites:/var/www/sites.local/d8/docroot/core$ sudo -u www-data SIMPLETEST_DB=mysql://root:pass@localhost/d8 SIMPLETEST_BASE_URL=http://sites.local ../../vendor/bin/phpunit -v /var/www/sites.local/d8/docroot/modules/contrib/workspace/tests/src/Functional/ReplicationSettingsTest.php
PHPUnit 4.8.26 by Sebastian Bergmann and contributors.

Runtime:	PHP 5.5.9-1ubuntu4.17
Configuration:	/var/www/sites.local/d8/docroot/core/phpunit.xml

E

Time: 30.85 seconds, Memory: 5.75MB

There was 1 error:

1) Drupal\Tests\workspace\Functional\ReplicationSettings::testReplicate
PHPUnit_Framework_Exception: Fatal error: Call to a member function click() on a non-object in /var/www/sites.local/d8/docroot/modules/contrib/workspace/tests/src/Functional/ReplicationSettingsTest.php on line 96

Caused by
exception 'ErrorException' with message 'unserialize(): Error at offset 0 of 185 bytes' in /var/www/sites.local/d8/vendor/phpunit/phpunit/src/Util/PHP.php:114
Stack trace:
#0 [internal function]: PHPUnit_Util_PHP->{closure}(8, 'unserialize(): ...', '/var/www/sites....', 114, Array)
#1 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/Util/PHP.php(114): unserialize('?Fatal error: C...')
#2 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/Util/PHP.php(52): PHPUnit_Util_PHP->processChildResult(Object(Drupal\Tests\workspace\Functional\ReplicationSettings), Object(PHPUnit_Framework_TestResult), '?Fatal error: C...', '')
#3 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/Framework/TestCase.php(722): PHPUnit_Util_PHP->runTestJob('<?php?if (!defi...', Object(Drupal\Tests\workspace\Functional\ReplicationSettings), Object(PHPUnit_Framework_TestResult))
#4 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/Framework/TestSuite.php(747): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#5 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(440): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#6 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/TextUI/Command.php(149): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#7 /var/www/sites.local/d8/vendor/phpunit/phpunit/src/TextUI/Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#8 /var/www/sites.local/d8/vendor/phpunit/phpunit/phpunit(47): PHPUnit_TextUI_Command::main()
#9 {main}
FAILURES!
Tests: 1, Assertions: 0, Errors: 1.
```